upgrading/upgrade-operator.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="upgrade-operator"]
+= Upgrading by using the Operator
+include::modules/common-attributes.adoc[]
+:context: upgrade-operator
+
+toc::[]
+
+[role="_abstract"]
+Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
+
+If you installed {product-title-short} using the Operator and selected *Automatic* in the *Update approval* field, {product-title-short} is automatically updated when a new software version is released. If you selected *Manual*, you must download new versions of the {product-title-short} Operator after a new software version is released and install them.
+
+If you chose automatic upgrades, and you want to roll back an upgrade, you must perform the steps described in the following sections.
```

Comment:
I think I was confused since the original use case was for people to "undo" automatic upgrades that had pushed a new version with defects? But I will rewrite!